script/cadence/contracts/FlowColdStorageProxy.cdc

```diff
@@ -44,9 +59,11 @@ pub contract FlowColdStorageProxy {
         pub var lastNonce: Int64
         access(self) let flowVault: @FungibleToken.Vault
         access(self) let publicKey: [UInt8]
+		access(self) let nodeDelegator: @FlowIDTableStaking.NodeDelegator
```

Comment:
Is this contract already deployed on a network? You aren't able to add fields in upgrades, so you'd need to find another way to do this if it is already deployed